src/Stratis.Bitcoin/Utilities/DBreezeSerializer.cs

```diff
@@ -11,11 +11,14 @@ namespace Stratis.Bitcoin.Utilities
     /// </summary>
     public class DBreezeSerializer
     {
+        public NetworkOptions NetworkOptions { get; private set; }
+
         /// <summary>
         /// Initializes custom serializers for DBreeze engine.
         /// </summary>
-        public void Initialize()
+        public void Initialize(NetworkOptions networkOptions = null)
```

Comment:
You are allowing networkOptions to be null but later in code you use it (though, in code that allows nulls). The problem is that this null would go all the way to the base class in order to be checked if it's null or not. I am not sure if it's a good idea architecture-wise.

Reply:
This is temporary.